pkg/tools/cli/loadtest/assets/loader.go

```diff
@@ -92,14 +92,14 @@ func Load(path string) (train, test, distances [][]float32, neighbors [][]int, d
 	}()
 	trainDim, v1, err := loadDataset(f, "train", loadFloat32)
 	if err != nil {
-		log.Error("couldn't load train dataset for path %s: %s", path, err)
+		log.Errorf("couldn't load train dataset for path %s: %s", path, err)
```

Comment:
Could you please teach me about this line?
I think log output is not necessary. And I may use `returns nil, nil, nil, 0, errors.Wrapf(err, "couldn't load train dataset for path %s: %s", path, err)` and the caller of this function outputs an error.

Reply:
I did not think much of it.
Certainly that is callee, so I think caller should have responsibility about logging. I will fix it.